config/locales/ca.yml

```
@@ -14,27 +31,41 @@ ca:
     previous: "Anterior"
     next: "Següent"
     download: "Descarregar:"
-    has_many_new: "Afegir %{model}"
+    has_many_new: "Afegir un altre %{model}"
     has_many_delete: "Eliminar"
     has_many_remove: "Treure"
+    move: "Moure"
     filters:
       buttons:
-        filter: "Filtrar"
-        clear: "Treure filtres"
+        filter: "Filtra"
+        clear: "Elimina els filtres"
+      predicates:
+        from: "Des de"
+        to: "Fins"
+    scopes:
+      all: "Tots"
+    search_status:
+      title: "Cerca activa"
+      title_with_scope: "Cerca activa per %{name}"
+      no_current_filters: "Sense filtres actius"
     status_tag:
       "yes": "Sí"
       "no": "No"
-      "unset": "No"
-    logout: "Desconnecta't"
+      "unset": "Desconegut"
+    logout: "Tanca la sessió"
     powered_by: "Powered by %{active_admin} %{version}"
     sidebars:
       filters: "Filtres"
+      search_status: "Estat de la cerca"
     pagination:
-      empty: "No hi ha %{model}"
-      one: "S'està mostrant <b>1</b> %{model}"
-      one_page: "S'estan mostrant <b>tots %{n}</b> %{model}"
-      multiple: "S'estan mostrant %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b> de <b>%{total}</b> en total"
-      multiple_without_total: "S'estan mostrant %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b>"
+      empty: "No s'ha trobat cap %{model}"
+      one: "S'està mostrant <b>1</b> de <b>1</b>"
+      one_page: "S'estan mostrant <b>tots %{n}</b>"
+      multiple: "Se n'estan mostrant <b>%{from}-%{to}</b> d'un total de <b>%{total}</b>"
+      multiple_without_total: "Se n'estan mostrant <b>%{from}-%{to}</b>"
```

This includes a required change to the `multiple` key since the form should be: "1-10 **of** 100" as an example. But since it overlaps with another question I had, I included another suggested change here.

I noticed the first part of each string is using a slightly different wording but it's all meant to be the same. So in the English version it's just "Showing 1..." as the prefix to keep it succinct. Mind you, I'm using Google Translate here, as while I'm fluent in Spanish, I don't know Catalan but am learning a bit from this so thank you.

Based on the translations I'm seeing in Google, I believe "S'està mostrant" would be the correct or rather intended one for all of these strings. Is there a reason though you choose not to use it?

```suggestion
      one: "S'està mostrant <b>1</b> de <b>1</b>"
      one_page: "S'està mostrant <b>tots %{n}</b>"
      multiple: "S'està mostrant <b>%{from}-%{to}</b> de <b>%{total}</b>"
      multiple_without_total: "S'està mostrant <b>%{from}-%{to}</b>"
```

---

Thank you for your detailed review. The original English wording "Showing xxx" produce strange, not natural, sentences in romance languages, starting with the use or gerunde, not a good choice in any romance language. However, the prevalence of English makes it less strange to newer generations.

I'm translating ActiveAdmin because it is  embedded into Muscat, that I have also translated and done some minor patches. And I have seen the original translation in the context of a real application and it doesn't look natural. So I have reviewed all other romance languages I'm familiared with (French, Italian, Portuguese and Spanish) to look for better ideas, but also this expresssion in other Catalan packages. I've also asked a Catalan philologist, and we've come out with this form. It uses this n, that is called a feeble pronoun, in the sense that it refers to (it stresses what we are referring to) whatever we are counting. Google translator is quite good, but it is not a native Catalan translator.

Summing up: I belive that this is the most natural translation I could find.

---

Thank you for clarifying. I appreciate it. What about the second suggestion changing "d'un total de" to "de" in the `multiple` key string?

---

This wording is an improvement of seeing a previous form (that I cannot recall now) that was not clear enough in a real Rails application that we are implementing (Muscat).  Now it is explicit about the total amount of elements in the list and what it is showing at this point.